code/tests/data-zones-debugging.qmd

```diff
@@ -24,7 +27,8 @@ file.exists(data_zones_zip_file)
 data_zones_exdir = "outputdata/DataZone_2024-07-02"
 unzip(data_zones_zip_file, exdir = data_zones_exdir)
 data_zone_files = list.files(data_zones_exdir, full.names = TRUE)
-length(data_zone_files) # 770, too few?
+length(data_zone_files) # 7457
+head(data_zone_files)
```

Comment:
Can you try reproducing this code @wangzhao0217?

I get the following:

```
[1] "outputdata/DataZone_2024-07-02/c(\"S01006802\", \"S01006802\").json"
[2] "outputdata/DataZone_2024-07-02/c(\"S01006806\", \"S01006806\").json"
[3] "outputdata/DataZone_2024-07-02/c(\"S01006807\", \"S01006807\").json"
[4] "outputdata/DataZone_2024-07-02/c(\"S01006808\", \"S01006808\").json"
[5] "outputdata/DataZone_2024-07-02/c(\"S01006809\", \"S01006809\").json"
[6] "outputdata/DataZone_2024-07-02/c(\"S01007208\", \"S01007208\").json"
```

Reply:
Update on this...

Reply:
Most of the file names are OK:

![image](https://github.com/nptscot/npt/assets/1825120/43913c43-577b-4e88-a9e0-0c4091cc54df)